distributed/spans.py

```diff
@@ -74,11 +74,12 @@ def span(*tags: str) -> Iterator[str]:
     if not tags:
         raise ValueError("Must specify at least one span tag")
 
-    prev_tags = dask.config.get("annotations.span.name", ())
+    annotation = dask.get_annotations().get("span")  # type: ignore
```

Comment:
```suggestion
    annotation = dask.get_annotations().get("span")
```
Looks like your pre-commit cache is obsolete. `rm -rf ~/.cache/pre-commit`

Reply:
mypy is still complaining in CI.